book/tut2_spec_pd_gpd.ipynb

```diff
@@ -0,0 +1,1002 @@
+{
```

Comment:
_Commented on notebook `book/tut2_spec_pd_gpd.ipynb` Cell 1 Line 5_
```markdown
# **Tutorial 2** - scientific Python ecosystem: `pandas` and `GeoPandas`

In this tutorial we will learn using
 1. [**pandas**](https://pandas.pydata.org/docs/): **tabular data** stored in ``pandas.DataFrame``s
 2. [**geopandas**](https://geopandas.org/en/stable/docs.html): **aspatial data** stored in ``geopandas.GeoDataFrame``s
```
TEST: Use "GeoPandas" instead of "geopandas"

Reply:
Done.